cpp/velox/utils/VeloxWriterUtils.cc

```diff
@@ -0,0 +1,114 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "utils/VeloxWriterUtils.h"
+
+#include <boost/algorithm/string.hpp>
+
+#include "config/GlutenConfig.h"
+#include "utils/ConfigExtractor.h"
+#include "utils/Exception.h"
+
+#include "velox/common/compression/Compression.h"
+#include "velox/type/Type.h"
+
+namespace gluten {
+
+using namespace facebook;
```

Comment:
`using namespace facebook::velox::parquet`, if no name conflict.
Then, we can remove the namespace qualifier: `facebook::velox::parquet::` in the following code.